src/components/ui/WithTooltip/index.tsx

```diff
@@ -11,12 +11,17 @@ export type ToolTipPositionType = 'top' | 'bottom' | 'right' | 'left';
 export interface Tooltipable {
   /* Put tooltips on any buttons */
   ['data-tooltip']?: boolean;
-  /* Tooltips locaiton on any buttons */
+  /* Tooltips location on any buttons */
   ['data-tooltip-position']?: ToolTipPositionType;
+  /* The element ID of an existing element to render tooltips into */
+  tooltipContainerId?: string;
+  /* Text content for tooltip. Falls back to the label */
+  tooltipContent?: React.ReactNode;
```

Comment:
Since, this is typed as `ReactNode` the description needs to change right? It will not be just Text content only right?